src/Rib/Extra/OpenGraph.hs

```diff
@@ -19,31 +24,76 @@ import qualified Text.URI as URI
 data OpenGraph
   = OpenGraph
       { _openGraph_title :: Text,
-        _openGraph_author :: Text,
+        _openGraph_url :: URI.URI,
```

Comment:
Shouldn't `og:url` be optional?

Reply:
Sure, I just figured any page using OpenGraph tags would have a URL.